developer/src/kmc-keyboard-info/test/fixtures/khmer_angkor/build/khmer_angkor.keyboard_info

```diff
@@ -15,13 +15,7 @@
         ]
       },
       "examples": [{
-        "keys": [
-          { "key": "x" },
-          { "key": "j" },
-          { "key": "m" },
-          { "key": "E" },
-          { "key": "r" }
-        ],
+        "keys": "x j m E r",
         "text": "\u1781\u17D2\u1798\u17C2\u179A",
```

Comment:
Why is this being reverted from #9621?  Also, how would this look in regard to #9671, and how would the issue be affected?

Reply:
I opted to go with the much shorter .kps format and not have two different formats for key strings.

For #9671, I don't really have a good answer, not sure what useful test to put in for that at present. The pattern is deliberately open-ended because we don't know what the key caps will have. Only four substrings have meaning:
* ` `: key delimiter, e.g. `x j m shift+e r`
* `+`: modifier joiner, e.g. `shift+ralt+z`, or `shift+3`.
* `space`: longhand for spacebar, e.g. `k a space m r`, and
* `plus`: <kbd>+</kbd> key, made available to avoid confusion with specs like `shift+plus`, which would otherwise be `shift++`.

These are not intended to be used to generate key events -- they are supposed to be human readable keys.